test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -113,6 +115,80 @@ PROTO_TOSTRING(UnaryOperator, op);
 /** Identifier (Name). */
 PROTO_TOSTRING(Name, name);
 
+std::string
+GetLuaMetatable()
+{
+	std::string retval;
+
+	retval += "metatable = {";
+
+	const char * const methods[] = {
+		"__add",
+		"__call",
+		"__concat",
+		"__div",
+		"__eq",
+		"__index",
+		"__le",
+		"__len",
+		"__lt",
+		"__mod",
+		"__mul",
+		"__newindex",
+		"__pow",
+		"__sub",
+		"__unm",
+	};
+
+	for (uint64_t i = 0; i < sizeof(methods) / sizeof(methods[1]); ++i) {
+		retval += methods[i];
+		retval += "=";
+		retval += "function()\n";
+		retval += "local retval = {};\n";
+		retval += "debug.setmetatable(retval, metatable)\n";
+		retval += "return retval; end";
+		retval += ",\n";
+	}
+
+	retval += "}\n";
```

Comment:
I suggest to use something like the following structure:
```lua
local metatable = {}
local function metamethod()
        return debug.setmetatable({}, metatable)
end 
metatable.__add = metamethod
-- ...
metatable.__unm = metamethod
```

It becomes much more easier to read. (AFAIKS you use the same approach in the last commit.)

Reply:
Fixed in `test/fuzz/luaL_loadbuffer/mt.lua`.